Aladin-iOS/Aladin-iOS/Screens/BookDetail/BookDetailVC.swift

```diff
@@ -0,0 +1,1000 @@
+        }
+        reviewDetail.snp.makeConstraints {
+            $0.top.equalTo(starStackView.snp.bottom).offset(10)
+            $0.centerX.equalToSuperview()
+        }
+        reviewButton.snp.makeConstraints {
+            $0.top.equalTo(reviewContainer.snp.bottom).offset(15)
+            $0.leading.trailing.equalToSuperview().inset(5)
+            $0.height.equalTo(37)
+        }
+        reviewRatingContainerDivier.snp.makeConstraints {
+            $0.top.equalTo(reviewButton.snp.bottom).offset(15)
+            $0.leading.trailing.equalToSuperview().inset(11)
+            $0.height.equalTo(1)
+        }
+        reviewTableView.snp.makeConstraints {
+            $0.top.equalTo(reviewRatingContainerDivier.snp.bottom).offset(15)
+            $0.leading.trailing.equalToSuperview()
+            $0.height.equalTo(282)
+        }
+        reviewContainerDivider.snp.makeConstraints {
+            $0.bottom.equalToSuperview()
+            $0.height.equalTo(4)
+            $0.trailing.leading.equalToSuperview()
+        }
+        
+        //MARK: - AladinInfoLayout
+        
+        aladinLogoView.snp.makeConstraints {
+            $0.top.equalToSuperview().offset(40)
+            $0.leading.equalToSuperview().offset(23)
+            $0.height.equalTo(32)
+            $0.width.equalTo(72)
+        }
+        aladinLogoLetterView.snp.makeConstraints {
+            $0.top.equalToSuperview().offset(51)
+            $0.leading.equalTo(aladinLogoView.snp.trailing).offset(9)
+            $0.height.equalTo(19)
+            $0.width.equalTo(54)
+        }
+        aladinCompanyLabel.snp.makeConstraints {
+            $0.top.equalTo(aladinLogoView.snp.bottom).offset(12)
+            $0.leading.equalToSuperview().offset(23)
+        }
+        aladinCompanyMoreButton.snp.makeConstraints {
+            $0.top.equalTo(aladinCompanyLabel)
+            $0.leading.equalTo(aladinCompanyLabel.snp.trailing).offset(4)
+            $0.height.width.equalTo(20)
+        }
+        serviceCenterLabel.snp.makeConstraints {
+            $0.top.equalTo(aladinCompanyLabel.snp.bottom).offset(20)
+            $0.leading.equalTo(aladinCompanyLabel)
+        }
+        buttonStackView.snp.makeConstraints {
+            $0.top.equalTo(serviceCenterLabel.snp.bottom).offset(12)
+            $0.centerX.equalToSuperview()
+            $0.leading.trailing.equalToSuperview().inset(23)
+        }
+        logoutButton.snp.makeConstraints {
+            $0.top.equalToSuperview()
+            $0.leading.equalToSuperview()
+            $0.height.equalTo(38)
+        }
+        inquiryButton.snp.makeConstraints {
+            $0.top.equalToSuperview()
+            $0.leading.equalTo(logoutButton.snp.trailing).offset(6)
+            $0.height.equalTo(38)
+        }
+        FAQButton.snp.makeConstraints {
+            $0.top.equalToSuperview()
+            $0.leading.equalTo(inquiryButton.snp.trailing).offset(6)
+            $0.trailing.equalToSuperview()
```

Comment:
```suggestion
$0.leading.trailing.equalToSuperview().inset(23)
```
<img width="350" alt="스크린샷 2022-11-23 18 24 35" src="https://user-images.githubusercontent.com/80062632/203511272-91774029-8049-4894-aa46-fecda3aaa26c.png">

버튼 레이아웃 요렇게 잡는게 더 나을거같아요